pkg/cmd/server/kubernetes/master.go

```diff
@@ -76,10 +76,12 @@ func (c *MasterConfig) RunPersistentVolumeClaimBinder() {
 }
 
 func (c *MasterConfig) RunPersistentVolumeClaimRecycler(recyclerImageName string, client *client.Client) {
+	privilged := true
```

Comment:
As I understand it, remount is necessary w/ the nfsv3 requirement.  Without remount or privileged server, this required nfs-ganehsa, another NFS server that makes this possible.

@rootfs can better explain, as this is his bash.  These bits are not my expertise.